crates/biome_js_analyze/tests/specs/nursery/noConstantMathMinMax/invalid.jsonc

```diff
@@ -0,0 +1,4 @@
+[
+	"Math.min(0, Math.max(100, x));",
```

Comment:
You might want to add more cases:
- `globalThis.Math`
- `window.Math`

Reply:
Good point 💡